yarn-project/pxe/src/simulator_oracle/index.ts

```diff
@@ -420,7 +427,8 @@ export class SimulatorOracle implements DBOracle {
     scopes?: AztecAddress[],
   ): Promise<Map<string, TxScopedL2Log[]>> {
```

Comment:
i felt like the complexity of this function was too high so I moved some of it to tagging_utils.ts + I did some naming changes were I found it confusing.

Reply:
I appreciate the code comments and renaming, but having to jump through 2 files makes it worse IMO. Also the fact that `WINDOW_HALF_SIZE` now lives in a different file but is imported here makes it harder to reason about the size of the window, I would at least make it something we provide to the utility functions so it' solely the responsibility of the caller to set.

Reply:
passing it as param makes sense. Will address 👍